frontend/appflowy_flutter/lib/workspace/presentation/home/home_stack.dart

```diff
@@ -280,6 +275,18 @@ class HomeTopBar extends StatelessWidget {
                     const SizedBox.shrink(),
               ),
             ),
+            BlocBuilder<PanesCubit, PanesState>(
+              builder: (context, state) {
+                return state.count > 1
+                    ? IconButton(
+                        onPressed: () {
+                          context.read<PanesCubit>().closePane(paneId);
+                        },
+                        icon: const Icon(Icons.close_sharp),
+                      )
+                    : const SizedBox.shrink();
```

Comment:
I prefer this, but it has same effect, if you prefer the other way just keep it as is :+1:
```suggestion
                if (state.count <= 0) {
                  return const SizedBox.shrink();
                }
                
                return IconButton(
                  onPressed: () => context.read<PanesCubit>().closePane(paneId),
                  icon: const Icon(Icons.close_sharp),
                );
```